docs/user_guide.md

```diff
@@ -39,108 +41,145 @@ Sample usage:
+Branch | The branch to analyse in the target repository
+Author's GitHub ID | GitHub ID of the target contributor in the repository
+Author's Display Name | Optional Field. The value of this field, if not empty, will be displayed in the dashboard instead of author's GitHub ID.
+[Optional] Author's Git Author Name | Detailed explanation below
+
+#### Git Author Name
+`Git Author Name` refers to the customizable author's display name set in the local `.gitconfig` file.
+It is displayed as author name as opposed to the `GitHub ID` in the entries.
+For example, in the Git Log's display:
 ```
 ...
 commit cd7f610e0becbdf331d5231887d8010a689f87c7
-Author: fakeAuthor <ma.tanghao@dhs.sg>
+Author: ConfiguredAuthorName <author@example.com>
 Date:   Fri Feb 9 19:14:41 2018 +0800
 
-    moved
+    Make some changes to show my new author's name
 
 commit e3f699fd4ef128eebce98d5b4e5b3bb06a512f49
-Author: harryggg <ma.tanghao@dhs.sg>
+Author: ActualGitHubId <author@example.com>
 Date:   Fri Feb 9 19:13:13 2018 +0800
 
-    new
+    Initial commit
  ...
 ```
-*fakeAuthor* and *harryggg* are both Local Git Author Name.
+**ActualGitHubId** and **ConfiguredAuthorName** are both `Git Author Name`.
 
-RepoSense assumes that authors' local Author Name is identical as their Github ID. However, it is not always the case. Many Git users will customize their local author name. Authors can use the following command to set the their local author name to Github ID before contributing:
+By default, Git uses the authors' `GitHub ID` as their `Git Author Name`.
+However, this is not always the case. Many Git users customize their `Git Author Name`.
+
+To fix this, authors can use the following command to reset their `Git Author Name` to `GitHub ID` before contributing:
 ```
-git config --global user.name “YOUR_GITHUB_ID_HERE”
+git config --global user.name “YOUR_GitHub_ID_HERE”
 ```
-If an author's local Git Author Name is not the same as his Github ID, he needs to fill in their local Author Name in the CSV config file. If more than one local Author Name is used, they can separate them with semicolon (；)
+For more details, do checkout this [FAQ](https://www.git-tower.com/learn/git/faq/change-author-name-email) on changing Git Author Identity.
 
-### Contribution Tags
-Although RepoSense's contribution analysis is quite accurate, authors can still use annotations to make sure that RepoSense correctly recognize their contribution. Special thanks to [Collate](https://github.com/se-edu/collate) for providing the inspiration for this functionality.
+If an author's `Git Author Name` is not the same as his `GitHub ID`, the `Git Author Name` needs to be filled into the CSV config file for accurate consolidation.
+If more than one `Git Author Name` is used, they can be separated by using the semicolon `;` operator within the `Git Author Name` column.
```

Comment:
Corrected this line for better clarity and fix a grammar error. Also, I think its good to add an example too.

If more than one Git Author Name are used, they can be separated by a semicolon ```;``` within the Git Author Name column. For example, ```Alice;Bob```.

Reply:
> For example, Alice;Bob

Since the above example uses **ActualGitHubId** and **ConfiguredAuthorName**, we should probably use that example here as well